### PR TITLE
adding handler and decoded to the event

### PR DESCRIPTION
**Pull request by glaslos**
Added the `handler` and `decoded` fields to the event and displaying it in the feed and content.

### CODE REVIEW

File: src/event.ts [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
     connKey?: number[];
     dstPort: number;
     rule?: string;
+    handler?: string;
     scanner?: string;
     sensorID: string;
     srcHost: string;
     srcPort: string;
     timestamp: string;
+    decoded?: any;
```
Comment (dkumiszhan) [inline src/event.ts:12]: Could you explain what these two fields represent? Should we add a description comment for each field in this file?

Comment (dkumiszhan) [inline src/event.ts:12]: Can this type be more specific?

Comment (glaslos) [reply src/event.ts:12]: Yes, that makes sense :+1:

Comment (glaslos) [reply src/event.ts:12]: They are specific to the protocol. Have a look at what you see in the consol on the web site

**Pull request merged**